bot.js

```diff
@@ -7,61 +7,50 @@ const bot = new Discord.Client();
-					const description = response[0].description ? response[0].description
-						: '*No description provided.*';
-					const embed = new Discord.RichEmbed();
-					embed.setTitle(response[0].title);
-					embed.setColor('#272727');
-					embed.setDescription(response[0].description);
-					embed.setURL(`https://codebottle.io/s/${response[0].id}`);
-					embed.setAuthor(response[0].username);
-					msg.channel.send(embed);
-				}).catch(e => {
-					msg.react('⛔');
-				});
-				break;
-			case 'get':
-				console.log(`=> Get: ${rest}`);
-				codebottle.get(rest).then(response => {
-					const description = response.description ? response.description
-						: '*No description provided.*';
-					const embed = new Discord.RichEmbed();
-					embed.setTitle(response.title);
-					embed.setColor('#272727');
-					embed.setDescription(response.description);
-					embed.setURL(`https://codebottle.io/s/${response.id}`);
-					embed.setAuthor(response.username);
-					msg.channel.send(embed);
-				}).catch(e => {
-					msg.react('⛔');
-				});
-				break;
-			case 'help':
-				const embed = new Discord.RichEmbed();
-				embed.setTitle('CodeBottle');
-				embed.setColor('#272727');
-				embed.setDescription(`
+	if (!msg.content.startsWith(prefix) || (msg.guild && !msg.channel.permissionsFor(bot.client.user).has('SEND_MESSAGES')))
+		return;
+
+	const command = msg.content.split(' ')[0].substring(1).toLowerCase().trim();
+	const rest = msg.content.substring(command.length + 2); // 1 for prefix + 1 for trailing space
+
+	switch(command) {
+		case 'search':
+			codebottle.search(rest).then(response => {
+				const description = response[0].description ? response[0].description : '*No description provided.*';
+				const embed = new Discord.RichEmbed()
+					.setTitle(response[0].title)
+					.setColor('#272727')
+					.setDescription(description)
+					.setURL(`https://codebottle.io/s/${response[0].id}`)
+					.setAuthor(response[0].username);
+				msg.channel.send(embed);
+			}).catch(e => msg.react('⛔'));
+			break;
+		case 'get':
+			codebottle.get(rest).then(response => {
+				const description = response.description ? response.description : '*No description provided.*';
+				const embed = new Discord.RichEmbed()
+					.setTitle(response.title)
+					.setColor('#272727')
+					.setDescription(response.description)
+					.setURL(`https://codebottle.io/s/${response.id}`)
+					.setAuthor(response.username);
+				msg.channel.send(embed);
+			}).catch(e => msg.react('⛔'));
+			break;
+		case 'help':
+			const embed = new Discord.RichEmbed()
+				.setTitle('CodeBottle')
+				.setColor('#272727')
+				.setDescription(`
 This is a bot for https://codebottle.io/
 
-\`!search <keywords>\` Search snippets with keywords
-\`!get <id>\` Get a snippet by its ID`);
-				msg.channel.send(embed);
-				break;
+\`!search <keywords>\` - Search snippets with keywords
+\`!get <id>\` - Get a snippet by its ID`);
+			msg.channel.send(embed);
+			break;
```

Comment:
Probably fix eslint config to allow the extra indent for switch cases

Reply:
1. I have no idea how to do ESLint stuff
2. I didn't touch the original indentation of switch cases. I simply removed a tab from the if statement

Reply:
oh ok